pkgs/applications/window-managers/sway/default.nix

```diff
@@ -34,7 +27,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     wayland libxkbcommon pcre json_c dbus libevdev
     pango cairo libinput libcap pam gdk_pixbuf
-    wlroots wayland-protocols
+    wlroots wayland-protocols swaybg
```

Comment:
Was there any reason to add `swaybg` to `buildInputs`? I haven't checked this for 1.1 but it wasn't required for 1.1-rc3 and doesn't make a difference.

Reply:
I read this:

> Unlike the other sway clients which have been split into separate projects, however, swaybg is a required dependency of sway.

In the release notes, and assumed that was actually the case, but turns out that's not the case. It builds without it.